b2sdk/file_lock.py

```diff
@@ -58,16 +62,26 @@ def as_dict(self):
             "unit": self.unit,
         }
 
+    def __repr__(self):
+        return '%s(%s %s)' % (self.__class__.__name__, self.duration, self.unit)
+
+    def __eq__(self, other):
+        if not isinstance(other, RetentionPeriod):
+            return NotImplemented
```

Comment:
remove it. If someone meets the interface but doesn't inherit from the class you expet him/her to, let it work. If not, it'll clearly crash one line later.

Reply:
I'll do that. I thought that the convention is to `return NotImplemented` so if someone compares this object to an `int` they don't get attribute errors and if someone wants to ducktype stuff they can implement their own `__eq__` and thanks to `return NotImplemented` be sure that their `__eq__` is always executed, regardless of the operand order.